src/web/app/pages-session/session-result-page/session-result-page.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, OnInit } from '@angular/core';
+
+/**
+ * Feedback session result page.
+ */
+@Component({
+  selector: 'tm-session-result-page',
+  templateUrl: './session-result-page.component.html',
+  styleUrls: ['./session-result-page.component.scss'],
+})
+export class SessionResultPageComponent implements OnInit {
```

Comment:
This is the page for student(registered or unregistered) to view result right? Instructor will have their own page `instructor-sessions-result-page`? Why not put it under `pages-student`?

Reply:
Because it's accessible to public with regkey (i.e. no need to join course), so strictly speaking it's not a "student" page. Similarly for the session submission page.